install/default_config.php

```diff
@@ -134,30 +134,79 @@
   'SMTP_USERNAME' => '',
   'STARTVIEW' => 'month.php',
   'SUMMARY_LENGTH' => '80',
-  'TABLEBG' => '#000',
-  'TEXTCOLOR' => '#000',
-  'THBG' => '#fff',
-  'THFG' => '#000',
+  'TABLEBG' => '#000000',
+  'TEXTCOLOR' => '#000000',
+  'THBG' => '#FFFFFF',
+  'THFG' => '#000000',
+  'TIMED_EVT_LEN' => 'D',
+  'TIMEZONE' => ini_get ( 'date.timezone' ),
   'TIME_FORMAT' => '12',
   'TIME_SLOTS' => '24',
   'TIME_SPACER' => '&raquo;&nbsp;',
-  'TIMED_EVT_LEN' => 'D',
-  'TIMEZONE' => 'America/New_York',
-  'TODAYCELLBG' => '#ff3',
+  'TODAYCELLBG' => '#FFFF33',
   'UAC_ENABLED' => 'N',
+  'UPCOMING_ALLOW_OVR' => 'N',
+  'UPCOMING_DISPLAY_CAT_ICONS' => 'Y',
+  'UPCOMING_DISPLAY_LAYERS' => 'N',
+  'UPCOMING_DISPLAY_LINKS' => 'Y',
+  'UPCOMING_DISPLAY_POPUPS' => 'Y',
+  'UPCOMING_EVENTS' => '0',
   'USER_PUBLISH_ENABLED' => 'Y',
   'USER_PUBLISH_RW_ENABLED' => 'Y',
   'USER_RSS_ENABLED' => 'N',
   'USER_SEES_ONLY_HIS_GROUPS' => 'Y',
   'USER_SORT_ORDER' => 'cal_lastname, cal_firstname',
   'WEBCAL_PROGRAM_VERSION' => 'v1.3.0',
-  'WEEK_START' => '0',
+  'WEEKENDBG' => '#D0D0D0',
   'WEEKEND_START' => '6',
-  'WEEKENDBG' => '#d0d0d0',
-  'WEEKNUMBER' => '#f63',
+  'WEEKNUMBER' => '#FF6633',
+  'WEEK_START' => '0',
   'WORK_DAY_END_HOUR' => '17',
-  'WORK_DAY_START_HOUR' => '8'
-  );
+  'WORK_DAY_START_HOUR' => '8',
+
+
+/* Things either not in the code at all yet,
+   or not user ajustable from "admin.php" or "prefs.php".
+   
+   Or, maybe, just need to be moved to a central location.
+   We shouldn't bundle stuff with WebCalendar that we don't maintain.
+   We should give users instructions on how to install if they want to use it,
+   with SUGGESTED versions. And put in the hooks to use it if we find it.
+
+  "/common/" directory is just for example purposes.
+  And it's outside WebCalendar directory structure; ie http://mysite.com/common/
+ */
+
+  'ACCESSIFY_FROM'         => '//yatil-cdn.s3.amazonaws.com/accessifyhtml5.min.js', # Put in screen reader compatible ARIA tags.
+  'CKEDITOR_FROM'          => '//cdn.ckeditor.com/4.6.0/basic/ckeditor.js',
+  'HKIT_FROM'              => '/common/hkit/',
+  'JQUERY_FROM'            => '//code.jquery.com/jquery-3.2.1.min.js integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"',
+  'JSCOOKMENU_FROM'        => '/common/JSCookMenu/',
+  'MODALBOX_FROM'          => '/common/modalbox/',
+  'NORMALIZE_CSS_FROM'     => '//necolas.github.io/normalize.css/7.0.0/normalize.css',
+  'PHPMAILER_FROM'         => '/common/phpmailer/',
+  'PROTOTYPEJS_FROM'       => '//ajax.googleapis.com/ajax/libs/prototype/1.7.3.0/prototype.js//ajax.googleapis.com/ajax/libs/prototype/1.7.3.0/prototype.js',
+  'RECAPTCHA_FROM'         => '//www.google.com/recaptcha/api.js',
+  'SCRIPTACULOUS_FROM'     => '/common/scriptaculous/',
+  'SMARTY_FROM'            => '/common/smarty/',
+  'TABCONTENT_FROM'        => '/common/tabcontent/',
+
```

Comment:
Most of the components above were removed from 1.9.0 with the move to composer.json.  I still see recaptcha and hkit are still in the code but not sure if anyone is using them.